arcgis-ios-sdk-samples/Maps/Set initial map location/SetInitialMapLocationViewController.swift

```diff
@@ -17,20 +17,18 @@ import ArcGIS
 
 class SetInitialMapLocationViewController: UIViewController {
     @IBOutlet weak var mapView: AGSMapView!
-    var map: AGSMap!
     
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        //add the source code button item to the right of navigation bar
+        // Add the source code button item to the right of navigation bar.
         (self.navigationItem.rightBarButtonItem as! SourceCodeBarButtonItem).filenames = ["SetInitialMapLocationViewController"]
         
-        //initialize map with `imagery with labels` basemap and an initial location
+        // Initialize map with imagery basemap and an initial location.
```

Comment:
This comment is no longer accurate as the map is no longer configured with a (custom) initial location (viewpoint).